nano/node/node.cpp

```diff
@@ -682,8 +675,8 @@ void nano::node::start ()
 			this_l->bootstrap_wallet ();
 		});
 	}
-	// Start port mapping if external address is not defined and TCP or UDP ports are enabled
-	if (config.external_address == boost::asio::ip::address_v6{}.any ().to_string () && (tcp_enabled || !flags.disable_udp))
+	// Start port mapping if external address is not defined and TCP ports are enabled
+	if (config.external_address == boost::asio::ip::address_v6{}.any ().to_string () && tcp_enabled)
```

Comment:
any() is a static function so we don't need to instantiate an address.

`boost::asio::ip::address_v6::any ()`

Reply:
https://github.com/nanocurrency/nano-node/pull/4132